shell/platform/android/io/flutter/embedding/engine/FlutterEngine.java

```diff
@@ -156,6 +156,7 @@ public FlutterEngine(@NonNull Context context) {
       this,
         androidLifecycle
     );
+    lifecycleChannel.appIsDetached();
```

Comment:
Are you hedging on the fact that Aaron is supporting channel buffers for this (since no entrypoint/binding is running by this point).

Reply:
I thought this will be end up stored in https://github.com/flutter/engine/blob/ca5642cad98079794882359526fc17c8cb861af3/lib/ui/hooks.dart#L116
if the binding hasn't initialized yet?

Reply:
Ah you're right, we had an existing single featured buffer here :)